views/layout.html

```diff
@@ -53,7 +53,7 @@
         Serve a different stylesheet for IE8 :
         http://www.theguardian.com/info/developer-blog/2013/oct/14/mobile-first-responsive-ie8 #}
         <!--[if gt IE 8]><!-->
-        <link rel="stylesheet" href="{{ asset('styles/main.css') }}?v=22">
+        <link rel="stylesheet" href="{{ asset('styles/main.css') }}?v=23">
```

Comment:
I think we need to delete the query string at the end (ex/ ?v=23)

Reply:
@JALWilliams We ended up needing the query string on the live site to stop the resources from caching too. Not sure how it ever worked previously, but I definitely needed it once I deployed to live, so just need to remember to increment value whenever we change js or css.